moto/dynamodb2/models/__init__.py

```diff
@@ -1690,6 +1720,17 @@ def restore_table_from_backup(self, target_table_name, backup_arn):
         self.tables[target_table_name] = new_table
         return new_table
 
+    def restore_table_to_point_in_time(self, target_table_name, source_table_name):
```

Comment:
Would you mind adding a doc string here to document the current limitations? Can be as simple as
```
"""
Currently only takes the source/target table arguments, and copies all items from the source
"""
```

Reply:
Done!